src/main/java/org/jboss/pnc/kafkastore/mapper/BuildStageRecordMapper.java

```diff
@@ -72,6 +72,10 @@ void initMetrics() {
 
                 buildStageRecord.setBuildStage(kafkaMessageDTO.getMdc().getProcessStageName());
                 buildStageRecord.setBuildId(buildId(kafkaMessageDTO.getMdc().getProcessContext()));
+
+                if (kafkaMessageDTO.getMdc().getProcessContextVariant() != null) {
```

Comment:
Should we put a default value of 0 in case the processContextVariant is null or empty? That would also mean patching the old values in the DB.

Reply:
I think @matejonnet said it could be any string, including null or empty. And from my understanding, we cannot assume it's an integer. In a lot of ways that check is redundant.

@matejonnet Let me know if I picked the wrong assumption!

Reply:
And thanks again for the review @vibe13 ! :)

Reply:
Yes, I can be any string.